apps/chat/src/types/publication.ts

```diff
@@ -0,0 +1,103 @@
+import { BackendDataNodeType, BackendResourceType } from './common';
+
+export enum PublicationFunctions {
+  EQUAL = 'EQUAL',
+  CONTAIN = 'CONTAIN',
+  REGEX = 'REGEX',
```

Comment:
According to Figma filters should be `Contains`, `Regex`, `Equal` and `Not contains`.
Why are they in uppercase here?
And how are you going to support `UserGroup` here?
![image](https://github.com/epam/ai-dial-chat/assets/20163971/61942c31-c0ee-4e10-b026-05b237323562)

Reply:
API doesn't support `Not contains` filter. Figma just represents how it should look like. For user groups, we can just add a new filter and pass it to the rules.

Reply:
Why are they in uppercase here?